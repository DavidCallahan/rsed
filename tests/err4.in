1
2
3 more
4
5 less
6
