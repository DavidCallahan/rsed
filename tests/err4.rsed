#!./rsed
print "Test 1"
skip to "more"
required not "more"
copy all
