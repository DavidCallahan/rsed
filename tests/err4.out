Test 1
input 3:  source 4: failed forbidde pattern: more
